setup.py

```diff
@@ -83,7 +83,11 @@ def build_extension(self, ext):
     description='Kompute: Blazing fast, mobile-enabled, asynchronous, and optimized for advanced GPU processing usecases.',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    ext_modules=[CMakeExtension('kp')],
+    ext_modules=[CMakeExtension('kp/kp')],
+    packages = find_packages(where="python/src"),
```

Comment:
Why is this being removed?

Reply:
just puts the module into a folder (kp), then __init__ imports kp.

was using it for testing if I needed to create a pure python class, I Implemented PyTypeContainer then moved it to c++.
It can be reverted back but really it just allows the ability to package pure python with the project if needed.